src/helpers/single-value-providers.ts

```
@@ -0,0 +1,65 @@
+    static createStaticIntegerValueProvider(staticValue: number): StaticIntegerValueProvider {
+        return new StaticIntegerValueProvider(staticValue);
+    }
+
+    static createRandomIntegerValueProvider(maxValue: number): RandomIntegerValueProvider {
+        return new RandomIntegerValueProvider(maxValue);
+    }
+
+    static createCustomFieldIntegerValueProvider(customField: string): CustomFieldIntegerValueProvider {
+        return new CustomFieldIntegerValueProvider(customField);
+    }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)